src/pages/Detail/Detail.js

```diff
@@ -54,13 +60,26 @@ const Detail = () => {
   window.addEventListener('scroll', onScroll);
 
   useOutSideClick(ref, () => setIsClicked(false));
+  useEffect(() => {
+    fetch(`${API.products}/3`, {
+      method: 'GET',
+    })
```

Comment:
```suggestion
    fetch(`${API.products}/3`)
```

method가 get인 경우 fetch의 두번쨰 인자인 option쪽은 defualt값이 get이므로 적어주지 않으셔도 괜찮습니다 😃